CHANGELOG.md

```diff
@@ -4,7 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [2.0.1] - 2017-09-22
+## [2.0.2] - 2017-09-25
```

Comment:
you've got 2 approvals, just don't forget to update this date before the merge and release